components/sensor/dlms_meter.rst

```diff
@@ -0,0 +1,208 @@
+DLMS_Meter Component
+====================
+
+.. seo::
+    :description: Instructions for setting up DLMS Meter component in ESPHome.
+    :image: dsmr.svg
+
+Component
+-------------
+
+The DLMS_Meter component connects to Smart Meters which use the encrypted DLMS/Cosem protocol over uart. These smartmeters are used for example everywhere here in Austria.
```

Comment:
remove the `here`: The DLMS_Meter component connects to Smart Meters which use the encrypted DLMS/Cosem protocol over uart. These smartmeters are used for example everywhere ~here~ in Austria.

Reply:
wow, thanks for the very quick review, will fix this.